.releaserc.json

```diff
@@ -0,0 +1,21 @@
+{
+  "branches": [{ "name": "master" }, { "name": "alpha", "prerelease": true }],
```

Comment:
Is it `master` or `main`? 😬😄

Reply:
master for this repo

Reply:
`master` should be fine - it's the same setup as other projects

Reply:
I'm happy to change it when we have one way to go.

Reply:
@warp-ds/warp-core-team I think we have a good mix of `main` and `master` as default branch in @warp-ds now. 🙈  What should we go with? (My vote is on `main`)

Reply:
I vote `main`, too. It's just one 1 syllable so I like it better 😄 Also: Warp = one syllable, vs Fabric = 2 syllables, sooooo
Old: Fabric-master
New: Warp-main
😄